plugins/woocommerce-blocks/assets/js/blocks/product-filters/inner-blocks/stock-filter/index.tsx

```diff
@@ -11,10 +11,12 @@ import { productFilterOptions } from '@woocommerce/icons';
 import './style.scss';
 import edit from './edit';
 import metadata from './block.json';
+import save from './save';
 
 if ( isExperimentalBlocksEnabled() ) {
 	registerBlockType( metadata, {
 		icon: productFilterOptions,
```

Comment:
We need to update the icon here to this: https://github.com/woocommerce/woocommerce/blob/fix%2Fproduct-filters-interactivity-api-various-fixes/plugins/woocommerce-blocks/assets/js/icons/library/product-filter-stock-status.tsx

Reply:
Fixed!